extensions/flatpack/deployment/src/main/java/org/apache/camel/quarkus/component/flatpack/deployment/FlatpackProcessor.java

```diff
@@ -36,8 +34,4 @@ NativeImageResourceBuildItem registerNativeImageResources() {
         return new NativeImageResourceBuildItem("net/sf/flatpack/xml/flatpack.dtd", "fpconvert.properties");
     }
 
-    @BuildStep
-    ReflectiveClassBuildItem registerReflectiveClasses() {
-        return new ReflectiveClassBuildItem(false, false, FlatpackException.class);
```

Comment:
Maybe we should keep this?

If it's a class provided by Camel, then I see no harm in the extension registering it for reflection. If the exception types are provided by the user application, then it seems reasonable that they need to register them for reflection manually.

Reply:
There is just the cost of embedding this class while it may not be used. But, yeah there is not real harm and that could prevent camel-quarkus user to register this camel class. I'm gonna drop this commit and push locally.
Many thanks all for review :+1: